src/esp/gfx/replay/Player.h

```diff
@@ -108,11 +108,17 @@ class Player {
   const std::vector<Keyframe>& debugGetKeyframes() const { return keyframes_; }
 
   /**
-   * @brief Applies a keyframe to the player.
+   * @brief Appends a Keyframe to the keyframe list.
    */
-  void applyKeyframe(const Keyframe& keyframe);
+  void pushKeyframe(const Keyframe& keyframe);
```

Comment:
`appendKeyframe` instead? I'm not sure `push` conveys adding to the back (stl has push_front, for example).